_posts/brianna-stedman/2020-09-02-brianna-first-reflection.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+author: brianna-stedman
+title: "Brianna's first reflection"
+---
+Hi there, hopefully this will work out. 
+
+I think I'm a big fan of Trinket! I really like how easy and integrated the exercises are with the course materials. I pulled up the Trinket Chapter 2 exercise in one tab, and 
+the Chapter 2 materials in the other tab after reading through it, and I found that pretty easy! I mentioned within my submission itself, but my biggest issue was concatenating
+the string "Hello" with the variable input of the name, so I used https://www.w3schools.com/python/gloss_python_string_concatenation.asp to help with the syntactical questions I had. 
+
+Github is something that I really need to get the hang of. This is where the majority of my lapses in vocabulary and understanding happen. I messed up last week's pull request, 
+and in trying to write this week's reflection post, I'm not 100% sure I actually know what I'm doing here. I feel pretty confident in the chapter exercises with Python, since there 
+is a dedicated place to input and submit those snippets of code. However, when it comes to actually editing the website... I'm using the "Github basics" step-by-step on the "How To" 
+page to have gotten this far, so that's really helpful, but not having a fundamental understanding of what pull requests actually are or how to "commit" things has made it really difficult. 
+
+Don't worry, I'm keeping track of the vocab I'm missing so hopefully we can cover it in class!
```

Comment:
Excellent. We'll keep using and defining these terms in class, and I promise they'll all make sense soon enough. Keep going in the meantime, as you have with this great post!